package.json

```diff
@@ -3,14 +3,18 @@
   "version": "1.2.1",
   "description": "server side rendering for tonic components",
   "main": "index.js",
+  "type": "module",
   "scripts": {
-    "test": "node test.js"
+    "test": "run-s test:*",
```

Comment:
I use this a lot for isolating individual steps but still logically grouping them. Happy to remove if we want to error on the side of less deps.